src/common/libkvs/kvs_txn_compact.c

```diff
@@ -210,7 +210,7 @@ int txn_compact (flux_kvs_txn_t *txn)
     }
 
     len = json_array_size (txn->ops);
-    if (!len)
+    if (!(len > 1))
```

Comment:
Good catch!

Slightly odd comparison there - what about `len < 2`?

Reply:
haha,yeah I guess that would be simpler :-)